scripts/install_dependencies/install_openvino_dependencies.sh

```diff
@@ -5,6 +5,27 @@
 
 set -e
 
+CMAKE_VERSION=3.13.0
+
+install_cmake() {
+    arch=$(uname -m)
+
+    if command -v apt-get &> /dev/null; then
```

Comment:
I think will be better to check if user already has a cmake and install only in case if cmake doesn't exist or version is old.
```
current_cmake_ver=$($cmake_command --version | sed -ne 's/[^0-9]*\(\([0-9]\.\)\{0,4\}[0-9][^.]\).*/\1/p')
required_cmake_ver=3.13.0
if [ ! @$(printf '%s\n' @$required_cmake_ver@ @$current_cmake_ver@ | sort -V | head -n1)@ = @$required_cmake_ver@ ]; then
```

Reply:
In general, I don't understand why do we need to increase cmake version in samples.
We don't use anything related to 3.13.

Moreover, I would even decrease cmake version to 3.7 to be able to compile samples on Debian 9 for Rasb PI

Reply:
@ilya-lavrenov Based on the information in [release-notes ](https://www.intel.com/content/www/us/en/developer/articles/release-notes/openvino-2022-3-lts-relnotes.html#:~:text=CMake*%203.13%C2%A0or%20higher) minimum required cmake version is 3.13.
Also build_samples.sh uses -S option (from [commit](https://github.com/openvinotoolkit/openvino/commit/36df7f51ae40b6b1a1a0dc6792ebcf8cd88f7a9c#:~:text=cmake%20%2DDCMAKE_BUILD_TYPE%3DRelease%20%2DS%20%22%24SAMPLES_PATH%22%20%2DB%20%22%24build_dir%22)), which is available since cmake 3.13.

Reply:
Can we revert `-S` option usage then?

Reply:
@ilya-lavrenov yes, in this case it would be better to revert  -S option usage and discard the changes from this PR.